hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -0,0 +1,1006 @@
+# Overview
+
+AST, also known as the analog sensor top, is the OpenTitan analog and
+security companion. Within AST are various analog functions (such as
+clocks, regulators, random number generators) needed to make the device
+function, as well as physical security sensors necessary to protect the
+device from physical attacks or manipulation.
+
+At a high level, AST communicates with a number of OpenTitan comportable
+modules. See diagram below.
+
+![](./media/image2.png)
+
+In the following sections, each family of connection is briefly
+described and explained. Note, the analog connections to AST are not
+shown in the diagram, but will be explained as well.
+
+# Interface Signals Table
+
+## Table notes
+
+### Signal naming conventions used in this document
+
+It complies with OpenTitan [<u>names
+</u>](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#naming)and
+[<u>suffixes</u>](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#suffixes)
+with some augmentations.
+
+  - > Clock signals start with clk\_\*

```

Comment:
i don't think you really need the `>` here

Reply:
OK